nep-27.mediawiki

```diff
@@ -0,0 +1,42 @@
+NEP: 27
+Title: Standardization of the Smallest Unit of GAS
+Author: Jimmy Liao <jimmy@r3e.network>
+Type: Standard
+Status: Draft
+Created: 2024-05-20
+
+==Abstract==
+
+This NEP proposes to standardize the terminology for the smallest unit of GAS by naming it "Datoshi." This change aims to eliminate ambiguity in NEO's financial transactions and further refine the clarity of GAS as a divisible currency unit.
+
+==Motivation==
+
+Currently, the term "GAS" refers to both the divisible cryptocurrency used for transactions on the NEO network and its smallest unit. This dual usage leads to confusion and inefficiencies in communication and documentation. By adopting "Datoshi" as the designated term for the smallest unit, this proposal seeks to enhance clarity and precision in the NEO ecosystem.
+
+==Specification==
+
+The term "Datoshi" is proposed as the name for the smallest unit of GAS. The relationship between GAS and Datoshi is defined as follows:
+- 1 Datoshi = 1e-8 GAS
+- 1 Kdatoshi (thousand Datoshi) = 1e-5 GAS
+- 1 Mdatoshi (million Datoshi) = 1e-2 GAS
+
+In equivalency:
+- 100,000,000 Datoshi = 100,000 Kdatoshi = 100 Mdatoshi = 1 GAS
+
+This specification introduces a clear, scalable denomination system for GAS that aids in transactions, contracts, and discussions within the NEO ecosystem.
+
+==Rationale==
+
+The choice of "Datoshi" was made after a community vote and discussion within the NEO Discord channels. The NEO Foundation (NF) has approved this terminology. This proposal aligns with the practices seen in other major blockchain projects, such as Bitcoin's smallest unit being named "Satoshi," which has been beneficial for transactional clarity and community engagement.
+
+==Backwards Compatibility==
+
+This proposal does not introduce any technical incompatibilities but will require updates to documentation, user interfaces, and educational materials to reflect the new terminology.
+
+==Test Cases==
+
+N/A
+
+==Implementation==
+
+Prior to final approval of this NEP, updates will be prepared for all NEO platform documentation and user interfaces to incorporate the term "Datoshi." This preparation will ensure a smooth transition upon formal acceptance of the proposal.
```

Comment:
We probably should update it everywhere before the standard merge, and then post a couple of links here.